asab/zookeeper/container.py

```diff
@@ -20,17 +20,14 @@ class ZooKeeperContainer(ConfigObject):
 	https://pypi.org/project/aiozk/
 	"""
 
-	def __init__(self, app, config_section_name, config=None):
+	def __init__(self, app, config_section_name, config=None, z_path=None):
```

Comment:
Add a comment about that alternatives, please.

Reply:
@ateska Can I merge?